### PR TITLE
Cloud server

### PR DESCRIPTION
**Pull request by linweilonger**
1. add the get_camera_angle api
2. refine swagger api (remove json format)  http://34.241.234.234:8000/swagger/
3. refine the cal Explosion location
4. auto cal and send the result to telegram

### CODE REVIEW

**Comment by cianmawhinney** [comment]
started having a quick look - looks really good! I'll finish off reviewing in the morning

File: bangnet_cloud/db_opera.py
Comment (cianmawhinney) [inline bangnet_cloud/db_opera.py:1]: Is this file just for testing? I take it there's nothing from this file used elsewhere?

File: bangnet_cloud/db.sqlite3
Comment (cianmawhinney) [inline bangnet_cloud/db.sqlite3:1]: Is this an old version of bangnet.db? Or is this different? I don't think I see any other references to it in your code

**Review by cianmawhinney — CHANGES_REQUESTED** [review]
Looks good! I think if the .pyc files and the .db files get removed then it looks good to merge.

File: bangnet_cloud/db_opera.py
Comment (linweilonger) [reply bangnet_cloud/db_opera.py:1]: yes，just for test, I can withdraw the file

File: bangnet_cloud/bangnet.db
[the remark this answers is not in the document]
Comment (linweilonger) [reply bangnet_cloud/bangnet.db:1]: This is a local database based on sqlite3, and all our data is stored here.

File: bangnet_cloud/db.sqlite3
Comment (linweilonger) [reply bangnet_cloud/db.sqlite3:1]: yes this file is invalid i will delete it

File: bangnet_cloud/__pycache__/cal_bang_location.cpython-38.pyc
[the remark this answers is not in the document]
Comment (linweilonger) [reply bangnet_cloud/__pycache__/cal_bang_location.cpython-38.pyc:1]: OK I will add .gitignore

**Review by cianmawhinney — APPROVED** [review]

**Pull request merged**